app/src/androidTest/java/org/kiwix/kiwixmobile/note/NoteFragmentTest.kt

```diff
@@ -18,18 +18,31 @@
 
 package org.kiwix.kiwixmobile.note
 
+import android.os.Build
 import androidx.test.internal.runner.junit4.statement.UiThreadStatement
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import org.junit.Before
 import org.junit.Test
 import org.kiwix.kiwixmobile.BaseActivityTest
 import org.kiwix.kiwixmobile.R
 
 class NoteFragmentTest : BaseActivityTest() {
+
+  @Before
+  fun waitForIdle() {
+    UiDevice.getInstance(InstrumentationRegistry.getInstrumentation()).waitForIdle()
+  }
```

Comment:
Instead of creating every subclass of `BaseActivityTest`. We should make an abstract function in `BaseActivityTest`.

Reply:
done